samples/scheduling/date-range-picker/form/src/index.ts

```diff
@@ -0,0 +1,34 @@
+import { defineComponents, IgcButtonComponent, IgcDateRangePickerComponent, IgcDialogComponent } from 'igniteui-webcomponents';
+import 'igniteui-webcomponents/themes/light/bootstrap.css';
+import './index.css'
+
+defineComponents(IgcDateRangePickerComponent, IgcButtonComponent, IgcDialogComponent);
+
+export class DateRangePickerForm {
+  constructor() {
+
+    document.getElementById('DateForm')?.addEventListener('submit', (event: SubmitEvent) => {
+      event.preventDefault();
+      showDialog(event.target as HTMLFormElement);
+    })
+
+    function showDialog(data: HTMLFormElement) {
+      console.log(data.DateRange.value)
+      var filePath = data.DateRange.value
+      const dialog = document.createElement('igc-dialog');
+      dialog.title = 'Range submission result';
+      dialog.addEventListener('igcClosed', () => dialog.remove());
+
+      const dump = document.createElement('pre');
+      dump.innerHTML = `<b>Start</b>: ${filePath.start} \n<b>End</b>: ${filePath.end}`;
```

Comment:
Optional: a name attribute can be assigned to the DRP, and the form data entries keys will be in the format "*name*-start" and "*name"-end".
Similar to how this is implemented in the Storybook Forms samples.
Also the filePath variable name does not seem suitable here.